samples/samples-java/src/main/java/com/function/AddProduct.java

```diff
@@ -0,0 +1,41 @@
+package com.function;
```

Comment:
Copyright for these

Can you create a follow up task to investigate linting options for Java?

Reply:
Tracking here: https://github.com/Azure/azure-functions-sql-extension/issues/462